### PR TITLE
Linkchips

### PR DESCRIPTION
**Pull request by RedSquirrelious**
This isn't quite working yet.  It removes the ticket from the recently visisted section in the hamburger menu.  But redux-persist does its job and keeps it after refresh.

### CODE REVIEW

File: src/reducers/ticketReducers.js [hunk 1]
```diff
@@ -45,6 +46,12 @@ const addIncidentsToState = (state, incidents) => {
   return newState
 }
 
+const removeTicketFromState = (state, ticketId) => {
+  let newState = { ...state }
+  let updatedState = _.omit(newState, ticketId)
```
Comment (macrobug) [inline src/reducers/ticketReducers.js]: No need to add a new dependency just for this one method. This ES6 method should do the trick:
```
let newState = { ...state }
Reflect.deleteProperty(newState, ticketId)
return newState
```

Comment (macrobug) [reply src/reducers/ticketReducers.js]: This should work too:
```
const { [ticketId]: toRemove, ...newState } = state
return newState
```

Comment (RedSquirrelious) [reply src/reducers/ticketReducers.js]: I'd looked at that one before and it seems like a good move.  Thanks!

File: src/components/elements/MenuLink.js [hunk 2]
```diff
@@ -12,7 +13,13 @@ const MenuLink = (type, id, onClick, dispatch) => {
       key={`${type}-${id}` }
       primaryText={<Link to={typeRoute} >{`${typeTitle} ${id}`}</Link>}
       rightIcon={<ActionDelete
-        onClick={() => dispatch(onClick(id)) } />}
+        onClick={
+          () => {
+            dispatch(onClick(id))
+            persistor.purge()
```
Comment (macrobug) [inline src/components/elements/MenuLink.js]: I'm not sure the MenuLink component is the best place for this logic. Maybe move to the reducer or action?

**Pull request closed without merging**